docs-index.html

```diff
@@ -91,7 +91,7 @@
         <link rel="stylesheet" href="docs-index.css" />
 
         <!-- Scripts -->
-        <script src="versions.js"></script>
+        <script src="versions.js" defer></script>
```

Comment:
Oops… this actually kills the page… presumably because by the time the `versions.js` file is loaded, the script further down on the page has already run.

<img width="1496" height="849" alt="image" src="https://github.com/user-attachments/assets/9c51815c-1eef-4094-af27-2920d8d662b5" />

Reply:
⚡